roles/galaxy_labs_engine/tasks/main.yml

```diff
@@ -90,6 +90,12 @@
     - permissions
     - update
 
+- name: Set temp dir permissions
```

Comment:
this should use permissions '02770' (set group ID with group rwx), owner and group should be galaxy.
Also, `labs_temp_dir` should not be in `/tmp', it should go into one of the galaxy temp directories (maybe `/mnt/tmp` which has other galaxy temp directories).

Reply:
@neoformit `group=101` would be better if possible, especially given you've confirmed it works.